compiler/rustc_feature/src/accepted.rs

```diff
@@ -273,6 +273,8 @@ declare_features! (
     /// Allows patterns with concurrent by-move and by-ref bindings.
     /// For example, you can write `Foo(a, ref b)` where `a` is by-move and `b` is by-ref.
     (accepted, move_ref_pattern, "1.48.0", Some(68354), None),
+    /// No longer treat an unsafe function as an unsafe block.
```

Comment:
Can you mention here that this is about a lint that still needs to be separately enabled?